src/commands/project/create.js

```diff
@@ -0,0 +1,78 @@
+const { flags } = require('@oclif/command')
+const { postProject } = require('../../requests/project')
+const {
+    getProjectIdentifierArg,
+    splitPathParams,
+    splitFlagParams
+} = require('../../support/command/parse-input')
+const BaseCommand = require('../../support/command/base-command')
+
+class CreateProjectCommand extends BaseCommand {
+
+    async run() {
+        const { args, flags } = this.parse(CreateProjectCommand)
+        const projectPath = getProjectIdentifierArg(args)
+        const [owner, name] = splitPathParams(projectPath)
+        const body = {
+            owner,
+            name
+        }
+        // Append optional flags to body if present
+        if (flags.description !== undefined) body.description = flags.description
+        if (flags.apis !== undefined) body.apis = splitFlagParams(flags.apis)
+        if (flags.domains !== undefined) body.domains = splitFlagParams(flags.domains)
+
+        await this.createProject(owner, name, JSON.stringify(body))
+    }
+
+    async createProject(owner, projectName, body) {
+        const createRequest = {
+            pathParams: [owner],
+            body
+        }
+
+        return this.executeHttp({
+            execute: () => postProject(createRequest),
+            onResolve: this.logCommandSuccess({ owner, projectName }),
+            options: {}
+        })
+    }
+}
+
+CreateProjectCommand.description = 'Creates a new project in SwaggerHub.'
+
+CreateProjectCommand.examples = [
+    'swaggerhub project:create organization/new_project_name --description "project description"',
+    'swaggerhub project:create organization/new_project_name -a "testapi1,testapi2"',
+    'swaggerhub project:create organization/new_project_name --apis "testapi1,testapi2"',
+    'swaggerhub project:create organization/new_project_name -d "testdomain3,testdomain4"',
+    'swaggerhub project:create organization/new_project_name --domains "testdomain3,testdomain4"',
+    // eslint-disable-next-line max-len
```

Comment:
Could you try shortening the example so that the eslint disabling isn't required? e.g. shorter description
Alternatively we can update /.eslint to `ignoreStrings` - https://eslint.org/docs/rules/max-len

Reply:
https://github.com/SmartBear/swaggerhub-cli/pull/228/commits/72b464d45dfa3f8ffd3fc02e95e3aee312acf199